app/src/main/java/com/omelan/cofi/MainActivity.kt

```diff
@@ -101,6 +102,25 @@ class MainActivity : MonetCompatActivity() {
         }
     }
 
+    private fun isAppInstalled(packageName: String): Boolean {
+        return try {
+            packageManager.getPackageInfo(packageName, 0)
+            true
+        } catch (e: Exception) {
+            false
+        }
+    }
```

Comment:
```suggestion
     fun isInstantApp(): Boolean {
        return try {
            packageManager.getPackageInfo(packageName, 0)
            false
        } catch (e: Exception) {
            true
        }
    }
```
I see no reason to pass packageName to the function as we would only ever check for `BuildConfig.APPLICATION_ID`
Also naming it `isInstantApp` makes it more clear in the place where we call it, why is this even needed